README.md

```diff
@@ -13,6 +14,8 @@ development environment. For example:
     <%= if Mix.env == :dev do %>
```

Comment:
I would probably remove this conditional here. I don't think we need to tell people to only show it in dev, that's up to them.